libc/src/math/generic/CMakeLists.txt

```diff
@@ -107,6 +111,9 @@ add_entrypoint_object(
   DEPENDS
     libc.src.__support.macros.properties.types
     libc.src.__support.FPUtil.nearest_integer_operations
+    libc.src.__support.macros.properties.architectures
```

Comment:
There are CMakeLists.txt files where `libc.src.__support.macros*` appear before `libc.src.__support.FPUtil.*` and files where it appears after. I assumed that it was because the header's dependencies are listed first and then the implementation file's, not because the ordering is random.